src/components/Snippet/SnippetTool/SnippetTool.js

```diff
@@ -1,19 +1,82 @@
+import { useState } from "react";
+import { useHistory } from "react-router";
 import styled from "styled-components";
 
 import Button from "../../common/Button";
+import SnippetSavingModal from "../../SnippetSavingModal";
+
+import { deleteSnippet } from "../../../api/service";
 
 import { TOOL } from "../../../constants/variants";
+import { OK } from "../../../constants/messages";
+
+import {
+  COPY,
+  SAVE,
+  SHARE,
+  DELETE
```

Comment:
Trailing commas가 누락되어 있네요!

```suggestion
  DELETE,
```